bulk-centralledger/chart-handler-bulk-transfer-fulfil/values.yaml

```diff
@@ -9,7 +9,7 @@ global: {}
 image:
   registry: "docker.io"
   repository: mojaloop/central-ledger
-  tag: v17.3.2
+  tag: v17.4.0-snapshot.7
```

Comment:
I suspect we'll need a release version after the dependant central-ledger PR is merged